vagga.yaml

```diff
@@ -24,3 +69,18 @@ commands:
     description: Shortcut to manage.py
     container: django
     run: [python3, manage.py]
+  run-postgres: !Supervise
+    description: Start the django development server using Postgres database
+    children:
+      app: !Command
+        container: django
+        environ:
+          DATABASE_URL: postgresql://vagga:vagga@127.0.0.1:5433/test
+        run: |
+            python3 manage.py migrate
```

Comment:
You should normally wait until PG becomes available to be able to apply migrations.